jitpack.yml

```
@@ -0,0 +1,7 @@
+before_install:
+  - rm -rf ./android/lib/src/main/res/raw/trust.js
+  - mkdir -pv ./android/lib/src/main/res/raw/
+  - git lfs fetch --all
+  - git lfs checkout dist/trust-min.js
+  - mv ./dist/trust-min.js ./dist/trust_min.js
+  - cp ./dist/trust_min.js ./android/lib/src/main/res/raw/
```

"-" doesn't work in filename?
```suggestion
  - cp ./dist/trust-min.js android/lib/src/main/res/raw/trust_min.js

```